orchagent/portsorch.h

```diff
@@ -216,6 +222,10 @@ class PortsOrch : public Orch, public Subject
     unordered_map<sai_object_id_t, int> m_portOidToIndex;
     map<string, uint32_t> m_port_ref_count;
     unordered_set<string> m_pendingPortSet;
+    uint32_t max_flood_control_types = 4;
```

Comment:
Should it be ```const```?
BTW, is there a way to get length of supported FC types instead of hard coding it?

Reply:
I don't think there is a way to obtain the size of enum or the last element. Let me know if you could think of a better approach.